setup.py

```diff
@@ -69,7 +69,7 @@ def build_extensions(self):
         'dimod/include/',
         ],
     install_requires=[
-        'numpy>=1.17.3,<2.0.0',  # keep synced with circle-ci, pyproject.toml
+        'numpy>=1.17.3,<2.0.0,!=1.21.0,!=1.21.1' # keep synced with circle-ci, pyproject.toml
```

Comment:
might want to add a `see https://github.com/dwavesystems/dimod/issues/901` comment, otherwise this is somewhat mysterious.

Reply:
done!